eds/blocks/hero/hero.js

```diff
@@ -25,30 +59,12 @@ export default function decorate(block) {
   videoSrc.setAttribute('type', 'video/mp4');
 
   if (videoElement) videoElement.append(videoSrc);
-  // if (videoAssets) block.prepend(videoElement);
-
-  const contentDiv = block.querySelector('div:last-child');
-  const contentChildDiv = contentDiv.querySelector('div');
-  const children = [...contentChildDiv.children];
-
-  const heroContent = domEl('div', { class: 'hero-content' });
-  const heroContentWrapper = domEl('div', { class: 'hero-content-wrapper' });
-  const heroFgImage = domEl('div', { class: 'hero-fg-image' });
-
-  // If there is no image, assume video is the foreground image
-  if (heroFgImage.children.length === 0) {
-    heroFgImage.appendChild(videoElement);
-  }
-
-  heroContent.prepend(heroContentWrapper);
-  block.prepend(heroContent, heroFgImage);
-
-  children.forEach((child) => {
-    if (!child.classList.contains('foreground-img')) {
-      heroContentWrapper.appendChild(child); // Move each child to left container except fg image
+  if (videoElement && videoAssets.length > 0) {
```

Comment:
this videoElement logic is a bit complex, (amongst other things, you're checking twice for the same conditions. I'm fine to merge as is, but after merge i might do a small sweep of this class, this logic can easily be simplified.

Reply:
For some reason it wasn't doing what I expected until I added the second check. Today, it's fine with just one.